CHANGELOG.md

```diff
@@ -65,6 +65,8 @@
 
 - [#3422](https://github.com/ChainSafe/forest/issues/3422) Add NV21 (Watermelon)
   support for calibration network.
+- [#3593](https://github.com/ChainSafe/forest/pull/3593): Add `--stateless` flag
```

Comment:
Is it worth elaborating a bit on what this flag exactly does?

Reply:
Done